dcc-portal-ui/app/scripts/common/js/services.js

```diff
@@ -143,23 +143,10 @@
     };
   });
 
-
-  // Prevent the settings service from being requested several times
-  var _settingsPromise = null;
-
   module.service('Settings', function (RestangularNoCache) {
-
+    _.extend(this, window.ICGC_SETTINGS);
```

Comment:
I would rather still keep the object encapsulated behind a `get()` call and avoid the `_.extend(this,...)`, in case we need to do a different strategy in the future.